### DISCUSSION TITLE
Timing points aren't saved in the editor

### DISCUSSION THREAD

**Discussion by Pennek** [ppy/osu#16399] [Q&A]
Even after placing a hit object and pressing `Ctrl` + `S`. This makes the editor practically useless for mapping.

version: 2022.108.0-lazer
video: https://www.youtube.com/watch?v=kOPKH4xKpiA

**Comment by peppy** [comment]
Can you please try placing a hitcircle after making changes and see if that helps? We don't have timing points supported for "change detection", but if you make another change it should save as expected.

**Reply by bdach** [reply]
the above is tracked at https://github.com/ppy/osu/issues/10314.

**Reply by Pennek** [reply]
I do exactly that in the video. It doesn't save.

**Reply by bdach** [reply]
[i wanna say this came up before.](https://github.com/ppy/osu/issues/14211)

the point is likely redundant, as in has no effect, because you didn't change any of the values.

**Reply by Pennek** [reply]
Well, that's very annoying because timing points are also used as markers for sections when mapping. Especially now that bookmarks don't exist

**Reply by bdach** [reply]
bookmarks will return. i haven't got round to it yet because that touches database in the current setup and i didn't want to start dredging that up before realm work is done.

maybe the timing screen should warn about timing points being redundant, though.

**Reply by Pennek** [reply]
Sometimes mappers want to set up timing points and adjust them later.
I would really prefer if they were kept even if redundant

**Reply by peppy** [reply]
I think eventually we'd want to show the redundant points on the list as such, for sure.

I think they would still be removed on save/load, though.
